test_runner/src/main.rs

```diff
@@ -1,3 +1,155 @@
+use core::time;
+use std::thread;
+
+use althea_kernel_interface::{KernelInterfaceError, KI};
+
 fn main() {
-    println!("Hello, world!");
+    /*
+    These are currently connected as such:
+    A---------B
```

Comment:
go ahead and move this namespace setup into a function call something arbitrary like "five_node_config". Over time we'll add different configs for different tests and eventually a function to generate random topologies